### PR TITLE
Add mozorg/about.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9827

### CODE REVIEW

**Review by peiying2 — CHANGES_REQUESTED** [review]
Overall looks good. A couple of things to call out:
- Removal of Taipei section, not sure how the layout will be affected.
- Need to have hoosteeno to get some clarity: not-for-profit or non-profit. We had issues with a few pages pushed out in the last days of the year because of conflicting info. We can keep them here as they are unless a decision was clearly written somewhere.

File: en/mozorg/about.ftl [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+about-firefox-fast-for-good = { -brand-name-firefox }: Fast for Good
+about-when-you-use-the-new-firefox = When you use the new { -brand-name-firefox }, you get a blazing fast experience while supporting { -brand-name-mozilla }’s mission to keep the internet healthy, weird and welcoming to all.
+about-walking-our-privacy-talk = Walking Our Privacy Talk
+about-when-the-facebook-breach-was = When the { -brand-name-facebook } breach was revealed, { -brand-name-mozilla } had an immediate response – and a { -brand-name-firefox } product to support user privacy.
+about-talking-internet-issues-irl = Talking Internet Issues IRL
+about-in-mozillas-irl-podcast-host = In { -brand-name-mozilla }’s IRL podcast, host Manoush Zomorodi shares real stories of life online and real talk about the future of the Web.
+about-corporation-foundation-not = Corporation. Foundation. Not-for-profit.
+about-learn-about-the-mozilla-foundation = Learn about the { -brand-name-mozilla-foundation }
+about-mozilla-puts-people-over-profit = { -brand-name-mozilla } puts people over profit in everything we say, build and do. In fact, there’s a non-profit Foundation at the heart of our enterprise.
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply en/mozorg/about.ftl:33]: This sentance relates specifically to the Foundation, so I think non-profit applies here?

File: en/mozorg/about.ftl [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+about-pioneers-of-the-open-web = Pioneers of The Open Web
+about-our-leadership-has-been-at = Our leadership has been at the forefront of building a healthier internet since Day 1. What began as an alternative to corporate domination has grown into a global force for good online.
+about-firefox-fast-for-good = { -brand-name-firefox }: Fast for Good
+about-when-you-use-the-new-firefox = When you use the new { -brand-name-firefox }, you get a blazing fast experience while supporting { -brand-name-mozilla }’s mission to keep the internet healthy, weird and welcoming to all.
+about-walking-our-privacy-talk = Walking Our Privacy Talk
+about-when-the-facebook-breach-was = When the { -brand-name-facebook } breach was revealed, { -brand-name-mozilla } had an immediate response – and a { -brand-name-firefox } product to support user privacy.
+about-talking-internet-issues-irl = Talking Internet Issues IRL
+about-in-mozillas-irl-podcast-host = In { -brand-name-mozilla }’s IRL podcast, host Manoush Zomorodi shares real stories of life online and real talk about the future of the Web.
+about-corporation-foundation-not = Corporation. Foundation. Not-for-profit.
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply en/mozorg/about.ftl:31]: Not-for-profit probably applies here as it covers both Corporation and Foundation

File: en/mozorg/about.ftl [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+# The number inside the strong tag will be big, bold, and on its own line. Remove the strong tag if you need to put the number in the middle of the phrase.
+about-2000-non-employee-guests-welcomed = <strong>2000</strong> non-employee guests welcomed each year
+
+about-berlin = Berlin
+
+# The number inside the strong tag will be big, bold, and on its own line. Remove the strong tag if you need to put the number in the middle of the phrase.
+about-500-annual-attendees-to-the = <strong>500</strong> annual attendees to the Berlin speaker series
+
+about-taipei = Taipei
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply en/mozorg/about.ftl:49]: Good call, I'll work to remove this from the page.

**Comment by alexgibson** [comment]
@peiying2 removed Taipei. Thank you for the review

**Review by peiying2 — APPROVED** [review]
Thank you for the fix. LGTM.

**Pull request merged**